src/animation/AnimationMixer.js

```diff
@@ -516,7 +517,7 @@ AnimationMixer.prototype = Object.assign( Object.create( EventDispatcher.prototy
 	// return an action for a clip optionally using a custom root target
 	// object (this method allocates a lot of dynamic memory in case a
 	// previously unknown clip/root combination is specified)
-	clipAction: function ( clip, optionalRoot ) {
+	clipAction: function ( clip, optionalRoot, blendMode = OverrideBlendMode ) {
```

Comment:
If an AnimationAction has a blendMode, then should an AnimationClip have that too?

Otherwise, if we were to implement loading additive animation in FBXLoader ([the FBX format supports this](https://help.autodesk.com/view/FBX/2017/ENU/?guid=__files_GUID_29727281_5732_4F89_A06A_BBADD4C8993F_htm)) the user would need to know which blendMode to use when creating actions from the clips returned by the loader. Similarly, the information would be lost when serializing animations to/from JSON.

Reply:
I could add it to AnimationClip as well and rather than default to override blending when creating an animation action, default to the blend mode stored on the clip.

Reply:
That would be my preference, but let's confirm with @mrdoob or @bhouston before you spend time on the change. :)